ytypes/list.go

```diff
@@ -83,15 +83,17 @@ func validateList(schema *yang.Entry, value interface{}) util.Errors {
 
 // checkKeys checks that the map key value for the list equals the value of the
 // key field(s) in the elements for the map value.
-//   entry is the schema for the list.
-//   structElems is the structure representing the element in the data tree.
-//   keyElems is the structure representing the map key in the data tree.
+//
+//	entry is the schema for the list.
+//	structElems is the structure representing the element in the data tree.
+//	keyElems is the structure representing the map key in the data tree.
+//
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.